### PR TITLE
Authorization with google on sign in

### PR DESCRIPTION
**Pull request by AJLambda**
# Description

Front end connects to back end on sign in
Google Auth working, need to implement tokens next

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [x] Tested locally with postman as well as incognito chrome window

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by Mrsteveson — APPROVED** [review]
Working for me locally. May want to include a little section in either readme regarding the .env variables. To get this running, create a .env file inside the client folder and include these variables sortve thing. REACT_APP_BE_URL=http://localhost:5000
REACT_APP_FE_ROOT=http://localhost:3000

File: client/src/App.js [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
+import Post from "./components/posts/Post";
+import CreatePost from "./components/posts/CreatePost.js";
+import { Button } from "reactstrap";
 
-import { getPosts, getUsers, getTest, getPost } from './actions';
-import SearchResults from './components/posts/SearchResults';
+import { getPosts, getUsers, getTest, getPost } from "./actions";
+import SearchResults from "./components/posts/SearchResults";
 
+import { getPosts, getUsers, getTest, getPost, login } from "./actions";
```
Comment (KevinBrack) [inline client/src/App.js:13]: Build preview is failing because these imports are duplicated on line 13 and 16.

**Review by KevinBrack — CHANGES_REQUESTED** [review]

File: client/src/actions/index.js [hunk 2]
```diff
@@ -64,60 +77,62 @@ export const addPost = (newPost) => dispatch => {
             dispatch({ type: ADD_FAILURE, payload: err });
         })
 }
+>>>>>>> 253800119084e8f89f377b1660783d7130e29f37
```
Comment (KevinBrack) [inline client/src/actions/index.js]: Function addPost is in here twice. Also these are merge marks showing the conflict has not finished resolving automatically. You can delete the merge marks and boil this down to one function, and that should work.

**Review by KevinBrack — CHANGES_REQUESTED** [review]

File: client/src/actions/index.js (hunk 2 above)
Comment (AJLambda) [reply client/src/actions/index.js]: Thank you, should be fixed now

**Review by Mrsteveson — APPROVED** [review]
Should be working fine now. Looked like we solved the issue causing the checks to fail.

**Review by Mrsteveson — APPROVED** [review]

**Review by KevinBrack — APPROVED** [review]

**Pull request merged**